README.rst

```diff
@@ -39,6 +40,35 @@ Configure Environment and Run
 
     dcos helloworld help
 
+
+Binary:
+-----------
+
+Use existing Binary:
+########################
+
+#. Download binary from github repo::
+
+   wget https://github.com/mesosphere/dcos-helloworld/raw/binary/binary/dcos-helloworld
```

Comment:
So we can use it in the helloworld universe example of how to specify cli binaries.